NAMESPACE

```diff
@@ -121,22 +117,33 @@ import(circlize)
 import(config)
 import(cowplot)
 import(data.table)
-import(dbplyr)
 import(digest)
 import(dplyr)
+import(forcats)
```

Comment:
which function was using forcats? Maybe we can rework it and avoid introducing this as dependency?

Reply:
Currently, the only package that is importing `forcats` is `prettyForestPlot`.